docs/user-guide/compute-resources.md

```diff
@@ -49,9 +49,12 @@ Documentation for other releases can be found at
 <!-- END MUNGE: GENERATED_TOC -->
 
 When specifying a [pod](pods.md), you can optionally specify how much CPU and memory (RAM) each
-container needs.  When containers have resource limits, the scheduler is able to make better
-decisions about which nodes to place pods on, and contention for resources can be handled in a
-consistent manner.
+container needs.  When containers have their resource requests and limits specified, the scheduleris
```

Comment:
Actually, better suggestion than "s/resource requests and limits/resource requests/" (I hadn't read your whole sentence) would be to say something like this:

When containers have their resource requests specified, the scheduler is able to make better decisions about which nodes to place pods on; and when containers have limits specified, contention for resources on a node can be handled in a specified manner."